test-lit/mm3.mod

```diff
@@ -1,4 +0,0 @@
-﻿!mod$ v1 sum:8d1e01df5c32946e
```

Comment:
It's not clear to me where this file was introduced but it should not have been.